tests/test_fixtures.py

```diff
@@ -28,15 +28,15 @@ class TestTemplate1(Template):
     def package(self):
         return {'name': 'package.name',
                 'version': 'package.version',
-                'license': 'package.license'}
+                'pkg_license': 'package.pkg_license'}
```

Comment:
Actually, if you could, can you rewrite this as `'pkg_license': 'package.license'`?

Reply:
This is a recently added feature (I'm continuing to work on it), and it would be neat to demonstrate that the value can be whatever people want to call `pkg_license`.

Reply:
Sure, if we want to keep 'pkg_license': 'package.license' here, we would need to have the same in tests/test_class_package.py", line 64, in testToDictTemplate
    self.assertEqual(dict1['package.pkg_license'] -changed to  self.assertEqual(dict1['package.license'] 
I will make the changes